VMBackup/main/Utils/ResourceDiskUtil.py

```diff
@@ -132,7 +132,7 @@ def get_resource_disk_mount_point(self,option=1): # pylint: disable=R0912,R0914
 			self.logger.log('unable to detect disk topology',True,'Error')
 
 		# device = "/dev/{0}".format(device)
-		partition = device + "1"  #assuming only one resourde disk partition
+		partition = "{0}{1}".format(device,"1")  #assuming only one resourde disk partition
```

Comment:
null check for device is required.

Reply:
get_resource_disk_mount_point method put a try-catch so that any exception inside the method shouldnt result in backup failure